statsmodels/tsa/exponential_smoothing/_ets_smooth.pyx

```diff
@@ -17,16 +17,30 @@ ctypedef fused numeric:
 cpdef _initialize_ets_smooth(
     numeric [:] params,
     numeric[:,:] xhat,
+    np.uint8_t [:] is_fixed,
```

Comment:
Might as well use np.long here which then maps to `dtype=int` when creating an array.  While uint8 is space-efficient, it is not any faster than using the native int type, and this mask is never large.

Reply:
I used `np.long_t` in the cython code and `dtype=np.long` now. Using `np.long_t` and `dtype=int` failed on the Windows machines.

Reply:
Still fails with `ValueError: Buffer dtype mismatch, expected 'long_t' but got 'long'` on the Windows machines. Any idea why?